app/controllers/playbook/pages_controller.rb

```diff
@@ -64,5 +65,12 @@ def set_kit
         redirect_to root_path, flash: { error: "That kit does not exist" }
       end
     end
+
+    def ensure_rails_kit_exists
+      # TODO: unsure why we cannot simply use the helpers that are included in ApplicationController - fix this
```

Comment:
@jasoncypret any idea why we can't use `Playbook::PbDocHelper` instead of re-creating this? I tried all kinds of things here and could not succeed.